contest/contributions/antoine-soetewey-how-to-create-a-simple-covid-19-dashboard.yml

```
@@ -0,0 +1,15 @@
+# Contribution's title, author, abstract
+title: How to create a simple Coronavirus dashboard specific to your country in R
+author: Antoine Soetewey
+abstract: |
+  This article will help you to build a visually appealing dashboard about the spread of COVID-19 Coronavirus specific to a country in R using flexdashboard.
+
+# Short name to be used in the website Gallery menu (can be the same as title)
+menu_entry: How to create a simple Coronavirus dashboard specific to your country in R
+# URL of the public source code repository
+repository: https://github.com/AntoineSoetewey/statsandr/blob/master/content/blog/2020-03-23-how-to-create-a-simple-coronavirus-dashboard-specific-to-your-country-in-r.html
+# Type of contribution and how it can be featured as gallery content
+# Keep only the type and content relevant to your contribution
+# For a Shiny app, provide its URL:
+type: webpage
+content: https://www.statsandr.com/blog/how-to-create-a-simple-coronavirus-dashboard-specific-to-your-country-in-r/ # e.g. my-app on shinyapps.io
```

Include acceptance `date`
```suggestion
content: https://www.statsandr.com/blog/how-to-create-a-simple-coronavirus-dashboard-specific-to-your-country-in-r/
date: 2020-05-22-13:30
```